### DISCUSSION TITLE
Cannot specify icoUrl for custom chains?

### DISCUSSION THREAD

**Discussion by linghuccc** [rainbow-me/rainbowkit#1311] [Q&A] [answered]
I was trying to add custom chains using rainbow kit by following the documentation in https://www.rainbowkit.com/docs/custom-chains. But an error was reported: iconUrl is not a valid parameter in Chain. Then I checked wagmi documentation on custom chains and found out that there is indeed no iconUrl in wagmi example. Does this mean we can not show any icon image for custom chains?

Dependency versions:
"@rainbow-me/rainbowkit": "^1.0.1",
"viem": "~0.3.35",
"wagmi": "~1.0.5"

Please advise. Thank you.

**Comment by DanielSinclair** [comment, marked as answer]
You'll want to use RainbowKit's `Chain` type which includes additional params.
```ts
import { Chain } from '@rainbow-me/rainbowkit';

const customChain: Chain = {
  ...
}
```

**Reply by DanielSinclair** [reply]
Have updated our docs for clarity.
